<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forminvoice1</class>
 <widget class="QWidget" name="Forminvoice1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="noinvoiceLabel">
      <property name="text">
       <string>noinvoice</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="noinvoiceLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="nopoLabel">
      <property name="text">
       <string>nopo</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nopoLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tglinvoiceLabel">
      <property name="text">
       <string>tglinvoice</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="tglinvoiceDateEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="diskonLabel">
      <property name="text">
       <string>diskon</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="diskonLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="ppnLabel">
      <property name="text">
       <string>ppn</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="ppnLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="jumlahbayarLabel">
      <property name="text">
       <string>jumlahbayar</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="jumlahbayarLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="idkaryawanLabel">
      <property name="text">
       <string>idkaryawan</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="idkaryawanLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="jangkapengirimanLabel">
      <property name="text">
       <string>jangkapengiriman</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="jangkapengirimanLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
